client/gutenberg/extensions/map/edit.js

```diff
@@ -206,10 +214,24 @@ class MapEdit extends Component {
 						<RawHTML>{ getAPIInstructions }</RawHTML>
 					</div>
 					<TextControl
-						placeholder="Paste Key Here"
+						className="components-text-control-api-key"
+						disabled={ apiRequestOutstanding }
+						placeholder={ __( 'Paste Token Here' ) }
 						value={ apiKeyControl }
 						onChange={ this.updateAPIKeyControl }
 					/>
+					<Button
+						className="components-text-control-api-key-submit"
+						isLarge
+						disabled={
+							! apiRequestOutstanding && ( apiKeyControl && apiKeyControl.length > 1 )
+								? false
+								: 'disabled'
```

Comment:
This should work:

```js
disabled={ ! apiRequestOutstanding && ( apiKeyControl && apiKeyControl.length > 1 ) }
```

Reply:
Yep, fixed in https://github.com/Automattic/wp-calypso/pull/28730/commits/13884f24a829357a36b3769999dbe3a580212e9b

Reply:
Actually, turns out the fix reversed the logic, making the field disabled when it should be enabled. Fixed in https://github.com/Automattic/wp-calypso/pull/28730/commits/b0dd7f6dc3a8577b3ee2a54af52136131b65c956